entries/w/wp.pl.json

```diff
@@ -0,0 +1,14 @@
+{
+  "WP": {
+    "domain": "wp.pl",
+    "url": "https://www.wp.pl/",
+    "tfa": [
+      "totp",
```

Comment:
Can you confirm that the TOTP option works with third party apps (such as Google Authentication, Authy, Duo, 1Password)?

Reply:
No, it doesn't. It's TOTP, but they changed the URI scheme to `olotpauth`.